content/blogposts/blog-post-starter-pack.md

```diff
@@ -0,0 +1,261 @@
+        <td>4.9</td>
+        <td>3.0</td>
+        <td>1.4</td>
+        <td>0.2</td>
+      </tr>
+    </tbody>
+  </table>
+</TableWrapper>
+
+
+### YouTube
+
+You can easily embed YouTube videos using the `YouTube` component. The `id` value is found at the end of the URL for any YouTube video. To display the title below the video, add the `showTitle` property.
+
+<YouTube
+  title="Accelerate developer onboarding with Sourcegraph" 
+  id="DgwvhRW1Cbc"
+  showTitle={true}
+/>
+
+
+### Video
+
+The `Video` component supports .mp4 and .webm files. First, upload the two versions of the video to Google Cloud. Then, add the root names of these files to the `source` property, without including their extensions. You can choose to repeat a video by including the `loop` property, or display the caption using the `showCaption` property.
+
+<Video 
+  source={{webm: 'blog/shift-left/01-code-reuse', mp4: 'blog/shift-left/01-code-reuse'}} 
+  loop={true}
+  title="Code reuse"
+  caption="An example of code search with Sourcegraph"
+  showCaption={true}
+/>
+
+
+### HubSpot Form Example
+
+Use the `HubSpotForm` component to drop a custom HubSpot form into your post. In most cases, the `masterFormName` and `chiliPiper` props are the only properties that need to be updated. Once added to your post, the HubSpot form will render in its place. All the options for this component are:
+
+- `formId` - an optional form ID if the form doesn't fall under one of our [master forms](https://docs.google.com/document/d/1qa393uAW_tjwNtH9jHf_d__PCJgmbsxNoxi9cO4XOcE/edit)
+- `masterFormName` - an optional master form name; `contactMulti`, `contactEmail`, `gatedMulti`, or `gatedEmail` for general contact forms and gated forms that have multi fields or an email field
+- `onFormSubmitted` - an optional callback function for triggering an action after a form is submitted
+- `inlineMessage` - an optional message to display after a form is submitted. We have a [generic fallback](https://github.com/sourcegraph/about/blob/main/src/components/HubSpotForm.tsx#L288) if none is provided. 
+- `chiliPiper` - an optional boolean (`true`/`false`) to enable Chili Piper. This is false by default.
+
+<HubSpotForm
+  masterFormName="contactEmail"
+  chiliPiper={true}
+/>
+
+## Code
+
+You can add syntax highlighting to any code you write! The Blog uses Prism for code formatting. Specify the language at the front of the code block. 
+
+**TypeScript**
+
+```ts
+export const BlogPost<BlogPostProps> = ({title, text}) = {
+  <h1>{title}</h1>
+  <p>{text}</p>
+  …
+}
+```
+
+
+## Links
+
+Add links with an anchor tag as you usually would: 
+
+<div className="text-center mb-5">
+  <a href="https://www.sourcegraph.com/search">Check out Sourcegraph Search!</a>
+</div>
+
+Instead of linking images, please link the image's caption. This will allow our Figure components to be more accessible to screen readers. To add a link to an image, provide a `caption` and `link` property to the `Figure` component.
```

Comment:
```suggestion
Instead of linking images using Markdown, please use the Figure component. This will allow our images to be more accessible to screen readers. To add a link to an image, provide a `caption` and `link` property to the `Figure` component.
```